.circleci/config.yml

```diff
@@ -300,6 +301,33 @@ commands:
             circleci tests glob "tests/**/*.spec.ts" | circleci tests split --split-by=timings > /tmp/tests-to-run
             yarn test:ci $(cat /tmp/tests-to-run)
           no_output_timeout: 15m
+      - run:
+          name: Create circleci_parameters.json
+          working_directory: ~/workbench/e2e
+          command: |
+            echo "{ \
+              \"circleBranch\": \"${CIRCLE_BRANCH}\", \
+              \"circleJob\": \"${CIRCLE_JOB}\", \
+              \"circleBuild\": \"${CIRCLE_BUILD_NUM}\", \
+              \"circleBuildUrl\": \"${CIRCLE_BUILD_URL}\", \
+              \"circleNodeIndex\": \"${CIRCLE_NODE_INDEX}\" \
+              }" \
+              > ./logs/circleci_parameters.json
+          when: always
+      - run:
+          name: Add CircleCI job data to test results json before upload json to storage bucket
+          working_directory: ~/workbench/e2e/logs
+          command: |
+            NEW_JSON=circleci-build-${CIRCLE_BUILD_NUM}-${CIRCLE_NODE_INDEX}-test-results.json
+            jq -s '.[0] * .[1]' test-results.json circleci_parameters.json > ${NEW_JSON}
+            # Remove all whitespaces
```

Comment:
Why? If you really need to do this, can you just pass the "-c" flag to fq instead?